api/testworkflows/v1/types.go

```diff
@@ -72,3 +72,17 @@ type PodConfig struct {
 	// volumes to include in the pod
 	Volumes []corev1.Volume `json:"volumes,omitempty" expr:"force"`
 }
+
+type Event struct {
+	Cronjob *CronJobConfig `json:"cronjob,omitempty"`
+}
+
+// cron job configuration
+type CronJobConfig struct {
+	// cron schedule to run a test workflow
+	Cron string `json:"cron" expr:"template"`
```

Comment:
Probably worth to allow passing optional `config` as well, that would be used to configure such TestWorkflow under this condition

Reply:
From what I got from discussion we can add it in next iteration